public/assets/images/CELO.svg

```diff
@@ -1,5 +1,6 @@
 <svg width="120" height="59" viewBox="0 0 120 59" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I recommend following the same file naming structure for assets in the public folder. In this case, these images should be named in `kebab-case`. this provides a more unified naming structure and all the URLs of assets on the page get to have the same structure as the pages

Reply:
it seems for this celo icon it didn't work out. usually if you are on mac os. changing the case of the characters is not considered as a change. you have then to change the name to something different stage the changes and then rename again the file to the name you want stage  & commit.